go/internal/feast/featurestore.go

```diff
@@ -92,7 +92,7 @@ func (fs *FeatureStore) GetOnlineFeatures(ctx context.Context, request *serving.
 
 	// TODO (Ly): Remove this BLOCK once odfv is supported
 	if len(requestedRequestFeatureViews)+len(requestedOnDemandFeatureViews) > 0 {
-		return nil, errors.New("ODFV is not supported in this iteration, please wait!")
+		return nil, errors.New("on demand feature views are currently not supported")
```

Comment:
re: errors,

grpc has a rich set of errors and status codes that can be super useful for callers so they can figure out whether the error is a server-side error or a client-side error. It can also help drive retries etc. 

https://pkg.go.dev/google.golang.org/grpc/status
https://www.grpc.io/docs/guides/error/
https://github.com/avinassh/grpc-errors/blob/master/go/server.go

Ideally, the there are error structs defined in `go/internal` and those errors are translated to grpc errors in `server.go`.

Reply:
sounds good, I'll take a look!

Reply:
alright I changed this specific error to use a grpc error; will change other ones in a future PR